doc/source/gallery/plot_clip.ipynb

```diff
@@ -1,236 +1,236 @@
+            "metadata": {
+                "tags": [
+                    "nbsphinx-thumbnail"
+                ]
+            },
+            "outputs": [],
+            "source": [
+                "chicago_clipped = chicago.clip(polygon)\n",
+                "\n",
+                "# Plot the clipped data\n",
+                "# The plot below shows the results of the clip function applied to the chicago\n",
+                "# sphinx_gallery_thumbnail_number = 2\n",
+                "fig, ax = plt.subplots(figsize=(12, 8))\n",
+                "chicago_clipped.plot(ax=ax, color=\"purple\")\n",
+                "chicago.boundary.plot(ax=ax)\n",
+                "poly_gdf.boundary.plot(ax=ax, color=\"red\")\n",
+                "ax.set_title(\"Chicago Clipped\", fontsize=20)\n",
+                "ax.set_axis_off()\n",
+                "plt.show()"
+            ]
+        },
+        {
+            "attachments": {},
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "<div class=\"alert alert-info\">\n",
+                "    \n",
+                "Note\n",
+                "\n",
+                "For historical reasons, the clip method is also available as a top-level function `geopandas.clip`.\n",
+                "It is recommended to use the method as the function may be deprecated in the future.\n",
+                "</div>\n",
+                "\n",
+                "Clip the groceries Data\n",
+                "----------------------\n",
+                "\n"
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": null,
+            "metadata": {},
+            "outputs": [],
+            "source": [
+                "groceries_clipped = groceries.clip(near_west_side)\n",
+                "\n",
+                "# Plot the clipped data\n",
+                "# The plot below shows the results of the clip function applied to the capital cities\n",
+                "fig, ax = plt.subplots(figsize=(12, 8))\n",
+                "groceries_clipped.plot(ax=ax, color=\"purple\")\n",
+                "near_west_side.boundary.plot(ax=ax, color=\"green\")\n",
+                "ax.set_title(\"Groceries Clipped\", fontsize=20)\n",
+                "ax.set_axis_off()\n",
+                "plt.show()"
+            ]
+        }
+    ],
+    "metadata": {
+        "kernelspec": {
+            "display_name": "Python 3",
+            "language": "python",
+            "name": "python3"
+        },
+        "language_info": {
+            "codemirror_mode": {
+                "name": "ipython",
+                "version": 3
+            },
+            "file_extension": ".py",
+            "mimetype": "text/x-python",
+            "name": "python",
+            "nbconvert_exporter": "python",
+            "pygments_lexer": "ipython3",
+            "version": "3.10.8"
+        }
+    },
+    "nbformat": 4,
+    "nbformat_minor": 4
+}
```

Comment:
double checking, this is just a whitespace change right?

Reply:
No clue, must have been pre-commit or something. I don't recall opening this file.